File: Backend/Entities/OrgTreeData.cs
using System.Collections.Generic;

namespace Backend.Entities
{
    public class OrgTreeData
    {
        public List<PersonRoleExtended> Roles { get; set; }
        public List<Job> Jobs { get; set; }
        public List<OrgGroupWithSupervisor> Groups { get; set; }
    }
}

File: Frontend/src/app/org-tree/org-tree-data.ts
import { OrgGroupWithSupervisor } from '../people/groups/org-group';
import { RoleExtended } from '../people/role';
import { Job } from '../job/job';

export class OrgTreeData {
  roles: RoleExtended[];
  jobs: Job[];
  groups: OrgGroupWithSupervisor[];
}
